data/defaultMagitechNetworkGameSetup.mgg

```diff
@@ -0,0 +1,92 @@
+Description=Classic - Newbie friendly
+MapFilterIndex=0
+Map=mountains
+Tileset=forest
+TechTree=magitech
+DefaultUnits=1
+DefaultResources=1
+DefaultVictoryConditions=1
+FogOfWar=1
+AdvancedIndex=0
+AllowObservers=1
+FlagTypes1=0
+EnableObserverModeAtEndGame=1
+AiAcceptSwitchTeamPercentChance=30
+FallbackCpuMultiplier=5
+PathFinderType=0
+EnableServerControlledAI=1
+NetworkFramePeriod=20
+NetworkPauseGameForLaggedClients=1
+FactionThisFactionIndex=0
+FactionCount=10
+NetworkAllowNativeLanguageTechtree=0
+FactionControlForIndex0=7
+ResourceMultiplierIndex0=5
+FactionTeamForIndex0=0
+FactionStartLocationForIndex0=0
+FactionTypeNameForIndex0=tech
+FactionPlayerNameForIndex0=???
+FactionPlayerUUIDForIndex0=
+FactionControlForIndex1=5
+ResourceMultiplierIndex1=5
+FactionTeamForIndex1=1
+FactionStartLocationForIndex1=1
+FactionTypeNameForIndex1=magic
+FactionPlayerNameForIndex1=???
+FactionPlayerUUIDForIndex1=
+FactionControlForIndex2=5
+ResourceMultiplierIndex2=0
+FactionTeamForIndex2=2
+FactionStartLocationForIndex2=2
+FactionTypeNameForIndex2=tech
+FactionPlayerNameForIndex2=???
+FactionPlayerUUIDForIndex2=
+FactionControlForIndex3=5
+ResourceMultiplierIndex3=5
+FactionTeamForIndex3=3
+FactionStartLocationForIndex3=3
+FactionTypeNameForIndex3=magic
+FactionPlayerNameForIndex3=???
+FactionPlayerUUIDForIndex3=
+FactionControlForIndex4=5
+ResourceMultiplierIndex4=5
+FactionTeamForIndex4=10
+FactionStartLocationForIndex4=4
+FactionTypeNameForIndex4=*Observer*
+FactionPlayerNameForIndex4=???
+FactionPlayerUUIDForIndex4=
+FactionControlForIndex5=5
+ResourceMultiplierIndex5=5
+FactionTeamForIndex5=10
+FactionStartLocationForIndex5=5
+FactionTypeNameForIndex5=*Observer*
+FactionPlayerNameForIndex5=???
+FactionPlayerUUIDForIndex5=
+FactionControlForIndex6=5
+ResourceMultiplierIndex6=5
+FactionTeamForIndex6=10
+FactionStartLocationForIndex6=6
+FactionTypeNameForIndex6=*Observer*
+FactionPlayerNameForIndex6=???
+FactionPlayerUUIDForIndex6=
+FactionControlForIndex7=5
+ResourceMultiplierIndex7=5
+FactionTeamForIndex7=10
+FactionStartLocationForIndex7=7
+FactionTypeNameForIndex7=*Observer*
+FactionPlayerNameForIndex7=???
+FactionPlayerUUIDForIndex7=
+FactionControlForIndex8=5
+ResourceMultiplierIndex8=5
+FactionTeamForIndex8=10
+FactionStartLocationForIndex8=8
+FactionTypeNameForIndex8=*Observer*
+FactionPlayerNameForIndex8=???
+FactionPlayerUUIDForIndex8=
+FactionControlForIndex9=5
+ResourceMultiplierIndex9=5
+FactionTeamForIndex9=10
+FactionStartLocationForIndex9=9
+FactionTypeNameForIndex9=*Observer*
+FactionPlayerNameForIndex9=???
+FactionPlayerUUIDForIndex9=
```

Comment:
@biels Thanks for submitting this. And could you add a short README to this directory to explain how someone wanting to run a headless server would use this file?